ziti/client.go

```diff
@@ -161,6 +161,11 @@ func (self *CtrlClient) GetSession(id string) (*rest_model.SessionDetail, error)
 		return nil, err
 	}
 
+	for _, er := range resp.Payload.Data.EdgeRouters {
```

Comment:
A comment on why in the code would be nice, but lgtm.

Reply:
comment added :+1: